src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2211,6 +2221,16 @@ def managed_instance_update(
     instance.proxy_override = (
         proxy_override or instance.proxy_override)
 
```

Comment:
Should you set `sku.name = None` here?

Reply:
I think it makes sense to null out the sku name here, because there is no requested sku name, and you don't yet know what the sku name will be (it will be found based on tier & family).

Reply:
Yes, it makes sense to set it to `None` here